.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+dist: trusty
+
+language: python
+
+python:
+    - "2.7"
+
+virtualenv:
+    system_site_packages: true
+
+services:
+    - cassandra
```

Comment:
Is it just much faster to include cassandra as a service rather than installing it in travis.sh, even with the downgrade?